ResultadosDigitais/hellper
# Insert the incident's commander  on database

### Description
This PR create a new column named commander in to hellper's database and add this column on database's controller.

### How to test?
On a local or staging environment, open an incident and verify if commander's column  is created with a value.

If you are going to  using the local environment for this test, first run this code on the   local database:
```
ALTER TABLE public.incident
  ADD commander text NULL;
```

### Expected behavior
Hellper must be able to create the incident  without a problem and atribute a value to the commander's column
